lighthouse-core/audits/cache-start-url.js

```diff
@@ -37,21 +37,18 @@ class CacheStartUrl extends Audit {
    * @return {!AuditResult}
    */
   static audit(artifacts) {
-    let cacheHasStartUrl = false;
-    const manifest = artifacts.Manifest.value;
-    const cacheContents = artifacts.CacheContents;
-
-    if (!(manifest && manifest.start_url && manifest.start_url.value)) {
+    if (!artifacts.Manifest || !artifacts.Manifest.value) {
```

Comment:
Since `artifacts.Manifest` is now `null` in the non-exist case, should we just change all these to this?

```js
if (artifacts.Manifest === null) {
```